### ISSUE TITLE
UI 0.14 - multi select export 

### ISSUE THREAD

**Issue by ikout** [blakeblackshear/frigate#12603]
**Describe what you are trying to accomplish and why in non technical terms**
At the review page when you right click and you select an event or detection at the top right you get the "export" button for that ONE selection.

With one event selected:
<img width="559" alt="image" src="https://github.com/user-attachments/assets/49ea8cdc-fab8-4599-b2cd-49cec6de5f3f">

**Describe the solution you'd like**
At the same page if you select MORE than one event the "export" option is "gone/hidden" which is frustrating if you need to export more than 2-3 events, so please add the option to export multiple individual events that have been selected...

Now if more than one is selected the "export" option is hidden/missing:
<img width="455" alt="image" src="https://github.com/user-attachments/assets/8741e783-a9b1-4840-a0da-b39e491881b4">

Additionally IF possible have the option as well for these selected events to be exported as one large clip and not as individual multiple ones.

**Comment by NickM-27** [comment]
The reason it is limited to one is because most users don't have systems with the resources to export multiple. We would need to build out a queue system

**Comment by ikout** [comment]
For the one large export option that could be possible heavy load task. For the multiple individual ones I think that a big part of the the work is there (unless an API is doing this) when you have the "unselect" option or the "delete".
Now that I'm thinking of this the same code used to delete the events could be modified to export them instead, I mean that has some kind of queue/list system already and instead of the delete command the export could be used.

In any case what the approach may be it will be useful if you go and have multiple events that you need to keep because something happened that day.

**Comment by ikout** [comment]
No, I did mean to do the exports in-parallel, multiple at the same time. Just a loop cycle and get the export one at a time.
In any case if that's not available and need some work that's fine, but it will be a useful feature to have in general.

**Comment by NickM-27** [comment]
> but it will be a useful feature to have in general.

sure, which is why the feature request is pinned. I was just explaining the reason why it was done that way

**Comment by 4423-coil** [comment]
I look forward to the implementation of this feature.
Currently, I export time ranges from 10 cameras individually. If I could export the same time range from multiple cameras in a single operation, it would significantly reduce the effort required.
Is there a planned implementation date?

**Comment by NickM-27** [comment]
We work on Frigate in our free time, there are no deadlines or planned releases
